ivy/data_classes/container/experimental/layers.py

```diff
@@ -1907,3 +1907,117 @@ def adaptive_avg_pool2d(
             prune_unapplied=prune_unapplied,
             map_sequences=map_sequences,
         )
+
+    @staticmethod
+    def static_ifftn(
+        x: ivy.Container,
+        s: Optional[Union[int, Tuple[int, ...]]] = None,
+        axes: Optional[Union[int, Tuple[int, ...]]] = None,
+        *,
+        norm: str = "backward",
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        prune_unapplied: bool = False,
+        map_sequences: bool = False,
+        out: Optional[ivy.Container] = None,
+    ):
+        """
+        ivy.Container static method variant of ivy.ifftn. 
```

Comment:
Seems like I made a wrong code suggestion 😅
```suggestion
        ivy.Container static method variant of ivy.ifftn. 
        
```